### DISCUSSION TITLE
Rest Query SSL Error

### DISCUSSION THREAD

**Discussion by jkatsabanissp** [Budibase/budibase#6849] [Help] [answered]
Hello,

Fiddling with Budibase on a DO droplet and attempting to connect to a REST API. The endpoint requires ignoring invalid or self-signed certificate errors similar to -k or -insecure in CURL.

**Comment by melohagan** [comment]
Hi @jkatsabanissp 

There is a similar discussion here: https://github.com/Budibase/budibase/discussions/6430
But this hasn't yet been investigated and resolved. 

Is this link of any use? https://www.digitalocean.com/community/questions/the-ssl-connection-could-not-be-established

**Reply by jkatsabanissp** [reply]
Thanks @melohagan, but I am trying to make Rest API calls through Budibase to a proprietary app that doesn't have valid certificates. Not sure what library BB is using to make API calls, but unless I can replicate something similar to Curl's "-Insecure" I'd have to find another way to import data. Will keep an eye out for possible solutions.

**Comment by melohagan** [comment, marked as answer]
Thanks for the detail @jkatsabanissp 

I have raised an issue here: https://github.com/Budibase/budibase/issues/6885
